options/create.go

```diff
@@ -37,6 +37,7 @@ const (
 // execution configuration, post-execution triggers, and output configuration.
 // It is not safe for concurrent access.
 type Create struct {
+	ID               string            `bson:"id" json:"id" yaml:"id"`
```

Comment:
I don't like the idea of a CreateOptions with an ID. What's the difference between a `Process.ID()` versus a `CreateOptions.ID`? Why does it even need an ID? It's for specifying options, and this seems to be a field which is externally injected but does nothing.

It's also deceptive - setting this for CreateOptions does nothing to a process ID - it only affects Command. This is a weird and unexpected decision and nobody will know how to use this field. I think it adds unnecessary complexity and is a dangerous road to go down to attach IDs everywhere. It's weird to mix CreateOptions, which are for processes, and Command when Process should be independent of Command.

This isn't worth adding just so you can an ID have in a log message from Command. Log the ID for the Command object instead.